content/en/install.md

```diff
@@ -3,101 +3,149 @@ title: Installation
 sidebar: false
 ---
 
-Installations methods include:
+The recommended method of installing SciPy depends on your preferred workflow.
+The common workflows can roughly be broken down into the following
+categories:
 
-- [Distributions](#distributions)
-- [pip](#pip-install)
-- [conda](#pip-install)
-- [Package Manager](#package_manager)
-- [Source](#source)
+- [Environment-based (e.g. `pip`, `conda`)](#environment-based) (the traditional workflow)
+- [Project-based (e.g. `uv`, `pixi`)](#project-based) (recommended for new users)
+- [System package managers](#system-package-managers) (simplest)
+- [Building from source](#building-from-source) (for debugging and development)
 
-Methods differ in ease of use, coverage, maintenance of old versions,
-system-wide versus local environment use, and control. With pip or
-Anaconda\'s conda, you can control the package versions for a specific
-project to prevent conflicts. Conda also controls non-Python packages,
-like MKL or HDF5. System package managers, like `apt-get`, install
-across the entire computer, often have older versions, and don\'t have
-as many available versions. Source compilation is much more difficult
-but is necessary for debugging and development. If you don\'t know which
-installation method you need or prefer, we recommend the Scientific
-Python Distribution [Anaconda](https://www.anaconda.com/download/).
+To install SciPy with [static type stubs],
+see [Installing with type stubs](#type-stubs).
 
-<a name="distributions"></a>
+[static type stubs]: https://typing.readthedocs.io/en/latest/guides/libraries.html
 
-# Scientific Python Distributions (recommended)
+<a name="environment-based"></a>
 
-Python distributions provide the language itself, along with the most
-commonly used packages and tools. These downloadable files require
-little configuration, work on almost all setups, and provide all the
-commonly used scientific Python tools.
+## Environment-based workflows
 
-[Anaconda](https://www.anaconda.com/download/) works on Windows, Mac,
-and Linux, provides over 1,500 Python packages, and is used by over 15
-million people. Anaconda is best suited to beginning users; it provides
-a large collection of libraries all in one.
+In environment-based workflows, you install packages into an environment, which you
+can activate and deactivate.
 
-For more advanced users who will need to install or upgrade regularly,
-[Mambaforge](https://github.com/conda-forge/miniforge#mambaforge) is a more
-suitable way to install the `conda` (and `mamba`, a faster `conda` alternative)
-package manager.
+If you have installed Python, you can create a virtual environment with a tool such
```

Comment:
"a tool such as" --- this suggests an external tool. I'd make explicit, e.g. "Python's built-in venv manager" or "Python's built-in venv tool".

Reply:
right, I was trying to also implicitly accommodate `virtualenv` and whatever other venv-like tools are used. But I agree that just mentioning `venv` here more clearly is probably sufficient.